app/dose-notifications/dose-notifications.ts

```diff
@@ -0,0 +1,34 @@
+import {Component} from "@angular/core";
+import {NavController, Alert} from 'ionic-angular';
+import {LocalNotifications} from 'ionic-native';
+
+export class DoseNotifications {
+
+    constructor(private navController: NavController) {
+        LocalNotifications.on("click", (notification, state) => {
+            let alert = Alert.create({
+                title: "Dose Amigos Medication Notification",
+                subTitle: "You currently have pending medications.",
+                buttons: ["OK"]
+            });
+            this.navController.present(alert);
+        });
+    }
```

Comment:
I was looking at this tutorial: https://www.thepolyglotdeveloper.com/2016/07/using-local-notifications-ionic-2-mobile-app/

And the constructor produces a pop-up when the user clicks on the alert that displays from the android pull down.